src/conf_mode/system-option.py

```diff
@@ -132,6 +132,11 @@ def apply(options):
     else:
       cmd('systemctl disable root-partition-auto-resize.service')
 
+    if 'enable-rfs-option' in options:
+        cmd('echo 32768 > /proc/sys/net/core/rps_sock_flow_entries')
```

Comment:
Can we use `sysctl -w`?
```
sysctl -w net.core.rps_sock_flow_entries=32768
```

Reply:
Can we use sysctl -w?

Reply:
I think it can, but sysctl -w will write to the sysctl config file putting